.azure-pipelines/MZoo-CI-Windows.yml

```diff
@@ -0,0 +1,32 @@
+trigger:
+- master
+
+jobs:
+- job: 'Test'
+  pool:
+    vmImage: 'vs2017-win2016'
+  strategy:
+    matrix:
+      py37:
+        python.version: '3.7'
+        onnx.version: '1.6'
+    maxParallel: 2
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      architecture: 'x64'
+
+  - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+    displayName: Add conda to PATH
+
+  - script: conda create --yes --quiet --name py$(python.version) -c conda-forge python=$(python.version) numpy protobuf
+    displayName: Create Anaconda environment
+
+  - script: |
+      call activate py$(python.version)
+      git lfs fetch --all
```

Comment:
this line fetches all the models in the model zoo. Does it make more sense to only fetch the changed models?

Reply:
yes

Reply:
Let's also add a check to validate this model using ort

Reply:
discussed offline -- including ORT validation in a future PR.